app/models/automation_request.rb

```diff
@@ -42,18 +42,17 @@ def self.create_from_ws(version, user, uri_parts, parameters, requester)
   end
 
   def self.create_from_scheduled_task(user, uri_parts, parameters)
-    [:namespace, :class_name].each { |key| uri_parts.delete(key) if uri_parts.key?(key) }
-    approval = {'auto_approve' => true}
-    uri_parts.stringify_keys!
-    parameters.stringify_keys!
-    create_from_ws("1.1", user, uri_parts, parameters, approval)
+    uri_parts = uri_parts.stringify_keys
+    parameters = parameters.stringify_keys
```

Comment:
By using `stringify_keys` without `!` we are dup'ing the objects at the top of the method so `uri_parts.delete` that was happening on the first line of the method before will no longer modify the object passed in.  👍